WordPress/build.gradle

```diff
@@ -126,7 +126,7 @@ dependencies {
     apt 'com.google.dagger:dagger-compiler:2.0.2'
     provided 'org.glassfish:javax.annotation:10.0-b28'
 
-    compile ('com.github.wordpress-mobile.WordPress-FluxC-Android:fluxc:develop-SNAPSHOT') {
+    compile ('com.github.wordpress-mobile.WordPress-FluxC-Android:fluxc:issue~fluxc-image-loader-SNAPSHOT') {
```

Comment:
This must be changed back to develop when https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/298 is merged